modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -142,9 +142,33 @@ properties:
           import_docs:
             type: boolean
             description: |
-              This property works in conjunction with the [enable_shared_bucket_access](#databases-foo_db-enable_shared_bucket_access) property. When **shared bucket access** is enabled, setting this property to true (`"import_docs": true`) indicates that this Sync Gateway node should perform import processing.
+              Introduced in Sync Gateway 1.5, this property specifies whether this Sync Gateway node should perform import processing.
+
+              This property works in conjunction with the [enable_shared_bucket_access](#databases-foo_db-enable_shared_bucket_access) property.
 
               `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
+
+              Starting Sync Gateway with `import_docs` enabled will generate the mobile-specific metadata for all the pre-existing documents in the Couchbase Server bucket. From then on, documents can be inserted on the Server directly (with N1QL or SDKs) or through the Sync Gateway REST API.
+
+
+              #### Import process
+
+              Import is the name for the process by which non-Sync Gateway writes obtain mobile metadata. Any non-Sync Gateway write is eligible for import.
+
+              The document is first run through the Sync Function to compute read security and routing, with the following differences:
+
+              - The import is processed with an admin user context in the Sync Function, similar to writes made through the Sync Gateway Admin API. This means that `requireAccess`, `requireUser` and `requireRole` calls in the Sync Function are treated as no-ops.
+              - During import, `oldDoc` is `nil` when the Sync Function is executed.
+
+              Only one node in a Sync Gateway cluster should be configured for feed-based import - it processes the full DCP stream.
+
+              The [Import+](#1.5/log) log key can be used to troubleshoot the import process further in the logs.
+
+              A filter function can be specified using the [databases.foo_db.import_filter](#1.5/databases-foo_db-import_filter) property to only import specific documents.
+
+              ##### Import and Sync Gateway Accelerator
```

Comment:
Remove this section. SG Accelerator is REMOVED and not a product offering. Please remove all references in docs